src/redux/reducers/experimentSettings/updateProcessingSettings.js

```diff
@@ -5,7 +5,19 @@ import initialState from './initialState';
 const updateProcessingSettings = (state, action) => {
   const { settingName, configChange } = action.payload;
 
-  const newConfig = _.cloneDeep(_.merge(state.processing[settingName], configChange));
+  const arrayMerge = (obj, src) => {
+    if (_.isArray(obj)) {
+      return src;
+    }
+  };
+
+  const newConfig = _.cloneDeep(
+    _.mergeWith(
+      state.processing[settingName],
+      configChange,
+      arrayMerge,
+    ),
+  );
```

Comment:
Change from `_.merge` to `_mergeWith` to handle array in config object. The default behaviour of `_.merge` will merge arrays in object, which means `_.merge({ a : [1]}, {a : [] })` results in `{a : [1]}`, while what is wanted is a replacement of the array : `{a : []}`

Reply:
so is `mergeWith` essentially equivalent to replacing the old config values for keys with the new values?